src/main/java/io/harness/cf/client/api/Config.java

```diff
@@ -12,7 +12,7 @@
 @Builder
 @AllArgsConstructor
 public class Config {
-  public static final int MAX_FREQUENCY = 60;
+  public static final int MAX_FREQUENCY = 5;
```

Comment:
I'm not sure if this should be called MIN instead of MAX.  Its used on line 42:
https://github.com/drone/ff-java-server-sdk/pull/1/files#diff-3d0133bafb5310c3bffc55c015e5b14f6115726415a481098d771b05d75f48e6R42

That function always returns the larger value of either MAX_FREQUENCY of that the user specifies.   That means if a user specifies 10,  MAX_FREQUENCY will be returned.   A user must set frequency to > MAX_FREQUENCY, i.e. user settings of 61 or greater will only ever superceed the MAX_FREQUENCY.